### PR TITLE
Pretransfer script and tests updated.

### PR DESCRIPTION
**Pull request by SilentFlame**
@unhammer 
Please have a look and provide feedback if anything more needed to be done.

### CODE REVIEW

File: apertium/apertium_pretransfer.cc [hunk 1]
```diff
@@ -181,10 +202,6 @@ void processStream(FILE *input, FILE *output, bool null_flush, bool surface_form
 void usage(char *progname)
 {
   wcerr << L"USAGE: " << basename(progname) << L" [input_file [output_file]]" << endl;
-  wcerr << L"  -n         assume no surface forms" << endl;
-  wcerr << L"  -e         treat ~ as compound separator" << endl;
-  wcerr << L"  -z         null-flushing output on '\0'" << endl;
-  wcerr << L"  -h         shows this message" << endl;
   exit(EXIT_FAILURE);
```
Comment (unhammer) [inline apertium/apertium_pretransfer.cc]: Could you put these back in?

File: apertium/apertium_pretransfer.cc [hunk 2]
```diff
@@ -36,27 +38,45 @@ using namespace std;
...
+        flag = 2;
+    }
     if(feof(input))
     {
       wcerr << L"ERROR: Unexpected EOF" << endl;
       exit(EXIT_FAILURE);
     }
     fputwc_unlocked(mychar, output);
+    if(flag == 1)
+      superblanks += mychar;
```
Comment (unhammer) [inline apertium/apertium_pretransfer.cc]: could you wrap this in a {}?

File: apertium/apertium_pretransfer.cc [hunk 3]
```diff
@@ -36,27 +38,45 @@ using namespace std;
...
+
     if(mychar == L'\\')
     {
       mychar = fgetwc(input);
       fputwc(mychar, output);
+      if(flag == 1)
+        superblanks += mychar;
```
Comment (unhammer) [inline apertium/apertium_pretransfer.cc]: {} here too

File: apertium/apertium_pretransfer.cc [hunk 4]
```diff
@@ -36,27 +38,45 @@ using namespace std;
...
     }
   }
+  if(flag == 1)
+    superblanks += L']';
```
Comment (unhammer) [inline apertium/apertium_pretransfer.cc]: {}

**Comment by SilentFlame** [comment]
@unhammer 
done the asked :)

File: tests/pretransfer/__init__.py [hunk 5]
```diff
@@ -89,6 +89,4 @@ class JoinGroupPretransferTest(PretransferTest):
 class InlineBlankPretransferTest(PretransferTest):
     inputs =          ["[{<i>}]^a<vblex><pres>+c<po># b$",          "[{<i>}]^a<vblex><pres>+c<po>+d<po># b$"]
     expectedOutputs = ["[{<i>}]^a# b<vblex><pres>$ [{<i>}]^c<po>$", "[{<i>}]^a# b<vblex><pres>$ [{<i>}]^c<po>$ [{<i>}]^d<po>$"]
-    @unittest.expectedFailure
-    def runTest(self):
-        super().runTest(self)
+
```
Comment (unhammer) [inline tests/pretransfer/__init__.py]: Could you add a test with input `[{<i>}]^a<vblex><pres>+c<po># b$ ^a<vblex><pres>+c<po># b$` expected `[{<i>}]^a# b<vblex><pres>$ [{<i>}]^c<po>$ ^a# b<vblex><pres>$ ^c<po>$` (ie. test that the following lexical unit does *not* get the inline blank belonging to the previous one)

File: apertium/apertium_pretransfer.cc [hunk 6]
```diff
@@ -12,14 +12,16 @@
  * General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+ * 02111-1307, USA.
```
Comment (unhammer) [inline apertium/apertium_pretransfer.cc]: Could you revert this change? referring to URL instead of snail mail is the new way :)

File: tests/pretransfer/__init__.py (hunk 5 above)
Comment (SilentFlame) [inline tests/pretransfer/__init__.py]: should I add this with a new class or append these to the present class InlineBlanktransferTest() ?

anyway in both cases the test is failing as of now.

Comment (unhammer) [inline tests/pretransfer/__init__.py]: new class I think

Comment (SilentFlame) [inline tests/pretransfer/__init__.py]: So, should I just add the class or do I need it to make pass the tests?

as that is not happening currently with this set of input and output.

Comment (SilentFlame) [inline tests/pretransfer/__init__.py]: new test class is now Passing, so have updated the script file with the mentioned changes.

- `superblanke` -> `wordblanks`
- tests now running without any failure.
